psd-web/app/controllers/investigations/coronavirus_related_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Investigations
+  class CoronavirusRelatedController < ApplicationController
+    def show
+      @investigation = Investigation.find_by!(pretty_id: params.require(:investigation_pretty_id)).decorate
+    end
+
+    def update
+      investigation = Investigation.find_by!(pretty_id: params.require(:investigation_pretty_id))
+
+      investigation.assign_attributes(params.require(:investigation).permit(:coronavirus_related))
+      create_audit = investigation.coronavirus_related_changed?
+
+      investigation.save!
+
+      AuditActivity::Investigation::UpdateCoronavirusStatus.from(investigation) if create_audit
```

Comment:
```suggestion
      if investigation.coronavirus_related_changed?
        investigation.save!
        AuditActivity::Investigation::UpdateCoronavirusStatus.from(investigation)
      end
```
Don't need to attempt to save if not changed.